README.rst

```diff
@@ -78,3 +78,24 @@ This will generate a new migration script. Then run:
 To apply the migration.
 
 For a full migration command reference, run ``python manage.py db --help``.
+
+
+Docs
+----
+
+To build the docs:
+
+move to the `slate` directory:
+::
+
+    cd slate/
+
+then build the docs with middleman:
+::
+
+    bundle exec middleman build
```

Comment:
Something about the submodule wasn't set up properly? Or I'm supposed to do something on my end to deal with it being a submodule?

I haven't worked with submodules much, do you know what I'm supposed to do?